### PR TITLE
CreateHash, DeduplicateValuesbyKey, JsonUnescape [Common Scripts Utility Consolidations]

### PR DESCRIPTION
**Pull request by samuelFain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5056
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5057
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5058

## Description
Community Pack Adoption for the following scripts:
-  CreateHash
- DeduplicateValuesbyKey
- JsonUnescape

Note: GuardRails fails because the "CreateHash" script uses md5 and sha1 hashing functions.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - The script CreateHash moved to CommonScripts pack.
       - The script DeduplicateValuesbyKey moved to CommonScripts pack.
       - The script JsonUnescape moved to CommonScripts pack.
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CreateHash/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "CreateHash",
     "description": "Simple wrapper of the https://docs.python.org/3/library/hashlib.html to produce sha512, sha256, sha1, md5 and Blake (simple hash) of a given text input",
     "support": "community",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.0.3",
     "author": "Joerg Stephan",
```
Comment (Shellyber) [inline Packs/CreateHash/pack_metadata.json:6]: Where is the deprecating notes in the files? + hidden: true

File: Packs/DeduplicateValuesbyKey/pack_metadata.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "DeduplicateValuesbyKey",
-    "description": "Given a list of objects (dictionaries) and a key (string), return a list of unique values that appear for that key in the list of objects. \n\nThe current implementation only works on the first-level of keys. That is, given a dictionary like {key1: {key2: value}}, you can return a list of unique values for key1, but not key2. ",
+    "name": "DeduplicateValuesbyKey (Deprecated)",
+    "description": "Deprecated. DeduplicateValuesbyKey script moved to CommonScripts pack.",
     "support": "community",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
```
Comment (Shellyber) [inline Packs/DeduplicateValuesbyKey/pack_metadata.json:5]: Add hidden: true

**Review by Shellyber — COMMENTED** [review]
Nice

File: Packs/CreateHash/pack_metadata.json (hunk 1 above)
Comment (samuelFain) [reply Packs/CreateHash/pack_metadata.json:6]: Added in later commit. all deprecating notes now exist for CreateHash.

Comment (Shellyber) [reply Packs/CreateHash/pack_metadata.json:6]: @samuelFain  I still don't see it in here. Where is the deprecated in the name and description in this file?

**Review by Shellyber — COMMENTED** [review]
Have a look at my note
Also it seems that guardrails fails on some errors, please have a look at it and fix accordingly

File: Packs/CreateHash/pack_metadata.json (hunk 1 above)
Comment (samuelFain) [reply Packs/CreateHash/pack_metadata.json:6]: Fixed now :)

**Review by Shellyber — APPROVED** [review]
Nice work

**Pull request merged**